docs/self-managed/concepts/multi-tenancy.md

```diff
@@ -94,3 +97,19 @@ Camunda 8 data produced from these resources will belong to the same tenant. The
 example on how tenant ownership is inherited.
 
 ![Tenant ownership inheritance diagram](img/multi-tenancy.png)
+
+### Standalone Zeebe usage
+
+For a standalone Zeebe installation, multi-tenancy is currently available with the following options:
+
+- Authentication enabled [through Identity](../../../self-managed/identity/what-is-identity/)
+- By [integrating an external tenant-managing component](../zeebe-deployment/zeebe-gateway/interceptors.md#implementing-a-tenant-providing-interceptor)
+  with Zeebe.
+
+Note that it's not possible to use multi-tenancy on the full Camunda 8 stack when
+[integrating an external tenant-managing component](../zeebe-deployment/zeebe-gateway/interceptors.md#implementing-a-tenant-providing-interceptor)
+in Zeebe, as the remaining Camunda 8 components don't support this setup.
+
+## Unsupported features
```

Comment:
This section is still not clear to me 😄 Which are the unsupported features ?

Reply:
It was clear to me that using the external tenant management will not work with Camunda 8 components.

Reply:
@nicpuppa that section is addressing the unsupported behavior of Camunda 8 as a complete product.

So we mention that multi-tenancy will only work if Identity is used and enabled.

I'm also expanding that section with [this PR](https://github.com/camunda/camunda-docs/pull/3101).

Reply:
It is ok for me, thanks for clarifying 👍